godot-core/src/registry/signal/typed_signal.rs

```diff
@@ -90,6 +90,59 @@ impl<'c, C: WithBaseField, Ps: meta::ParamTuple> TypedSignal<'c, C, Ps> {
         self.owner.to_owned()
     }
 
+    /// Fully customizable connection setup.
+    ///
+    /// The returned builder provides several methods to configure how to connect the signal. It needs to be finalized with a call to
+    /// [`ConnectBuilder::done()`].
+    pub fn connect_builder(&mut self) -> ConnectBuilder<'_, 'c, C, (), Ps, ()> {
+        ConnectBuilder::new(self)
+    }
+
+    /// Directly connect a Rust callable `godot_fn`, with a name based on `F`.
+    ///
+    /// This exists as a short-hand for the connect methods on [`TypedSignal`] and avoids the generic instantiation of the full-blown
+    /// type state builder for simple + common connections, thus hopefully being a tiny bit lighter on compile times.
+    fn inner_connect_godot_fn<F>(
+        &mut self,
+        godot_fn: impl FnMut(&[&Variant]) -> Result<Variant, ()> + 'static,
+    ) {
+        let callable_name = make_callable_name::<F>();
+        let callable = Callable::from_local_fn(&callable_name, godot_fn);
+
+        let signal_name = self.name.as_ref();
+        self.owner.with_object_mut(|obj| {
+            obj.connect(signal_name, &callable);
+        });
+    }
+
+    /// Connect an untyped callable, with optional flags.
+    ///
+    /// Used by [`ConnectBuilder::done()`]. Any other type-state (such as thread-local/sync, callable debug name, etc.) are baked into
+    /// `callable` and thus type-erased into runtime logic.
+    pub(super) fn inner_connect_untyped(
+        &mut self,
+        callable: &Callable,
+        flags: Option<ConnectFlags>,
+    ) {
+        use crate::obj::EngineBitfield;
+
+        let signal_name = self.name.as_ref();
+
+        self.owner.with_object_mut(|obj| {
+            let mut c = obj.connect_ex(signal_name, callable);
+            if let Some(flags) = flags {
+                c = c.flags(flags.ord() as u32);
+            }
+            c.done();
+        });
+    }
+
+    pub(crate) fn to_untyped(&self) -> crate::builtin::Signal {
+        crate::builtin::Signal::from_object_signal(&self.receiver_object(), &*self.name)
+    }
+}
```

Comment:
Moving those up changes the order of declaration, which no longer documents the symbols in order of importance.

See
- https://godot-rust.github.io/docs/gdext/pr-1042/godot/register/struct.TypedSignal.html#implementations
- https://godot-rust.github.io/docs/gdext/master/godot/register/struct.TypedSignal.html#implementations

Could you restore the order?